Extension/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # C/C++ for Visual Studio Code Changelog
 
+## Version 1.22.3: September 16, 2024
```

Comment:
Is this date correct? If so, why is the changelog being updated now?

Reply:
i.e. can we release Thursday or did we want to wait till Monday for some reason?